sample/simple-videoplayer-channel/.vscode/launch.json

```diff
@@ -0,0 +1,13 @@
+{
+    "version": "1.0.0",
```

Comment:
if we are commiting launc.json, let's push settings.json as well with dummyvalues, similar to the screenshot i posted.